conda-env/ci.yml

```diff
@@ -22,11 +22,16 @@ dependencies:
   - mache >=0.15.0
   - matplotlib-base
   - netcdf4
-  - numpy
-  - xarray
+  - numpy >=1.23.0
+  - xarray >=2023.02.0
```

Comment:
I constrained `numpy >=1.23.0` because that one supports Python 3.11 (future-proofing).
I constrained `xarray >=2023.02.0` because that one drops Python 3.8 support.